build-llvm34.md

```diff
@@ -6,32 +6,23 @@ slug: build-llvm34
 ---
 
 The current procedure for building KLEE with LLVM 3.4 (recommended) is outlined below.
+These instructions use KLEE's new CMake based build system. If you wish to build with
+KLEE's older build system (not recommended) then see [here]({{site.baseurl}}/build-llvm34-legacy).
```

Comment:
Say autoconf here, drop "not recommended", as it's a bit strong, we're still maintaining the autoconf system.